src/main/java/com/bgsoftware/superiorskyblock/core/menu/button/impl/menu/BiomeButton.java

```diff
@@ -45,6 +48,7 @@ private BiomeButton(TemplateItem buttonItem, GameSound clickSound, List<String>
         this.lackPermissionCommands = lackPermissionCommands == null ? Collections.emptyList() :
                 Collections.unmodifiableList(lackPermissionCommands);
         this.biome = biome;
+        this.cfg = YamlConfiguration.loadConfiguration(new File(plugin.getDataFolder(), "menus/biomes.yml"));
```

Comment:
Do not load the config here.
The entire interaction with the menu is done inside the biomes-menu implementation, and it should stay there only.